plugin/src/main/java/io/snyk/eclipse/plugin/views/snyktoolview/SummaryBrowserHandler.java

```diff
@@ -0,0 +1,61 @@
+package io.snyk.eclipse.plugin.views.snyktoolview;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.swt.browser.Browser;
+import org.eclipse.swt.browser.BrowserFunction;
+
+import io.snyk.eclipse.plugin.html.StaticPageHtmlProvider;
+import io.snyk.eclipse.plugin.preferences.Preferences;
+import io.snyk.languageserver.protocolextension.SnykExtendedLanguageClient;
+
+public class SummaryBrowserHandler {
+	private Browser browser;
+
+	public SummaryBrowserHandler(Browser browser) {
+		this.browser = browser;
+	}
+
+	public void initialize() {
+
+		new BrowserFunction(browser, "totalIssues") {
+			@Override
+			public Object function(Object[] arguments) {
+				Preferences.getInstance().store(Preferences.ENABLE_DELTA, Boolean.FALSE.toString());
+				updateConfiguration();
```

Comment:
This could be extracted to a method that takes a boolean parameter to avoid duplication.